client/app/app.js

```diff
@@ -211,4 +205,17 @@ angular
   .run(($translate) => {
     moment.locale(_.first($translate.use().split('_')));
   })
+  .run((constants, atInternet, OvhApiMe) => {
```

Comment:
Explicit annotation would be nice.

```suggestion
  .run(/* @ngInject */(constants, atInternet, OvhApiMe) => {
```